mypy.ini

```diff
@@ -76,9 +74,6 @@ check_untyped_defs = False
 disallow_untyped_calls = False
 disallow_untyped_defs = False
 
-[mypy-graphql_compiler.compiler.helpers.*]
-disallow_untyped_calls = False
```

Comment:
This is perhaps my favorite part about `typing-copilot` — it catches and tightens settings I never would have thought about doing by hand. In this case, it noticed that `graphql_compiler.compiler.helpers` was calling a previously-untyped function that is now typed.

Since this suppression is no longer necessary, it is removed. That module is guaranteed to now and forever more only call fully-typed code.